### PR TITLE
use auth0/userinfo

### PR DESCRIPTION
**Pull request by l-vincent-l**
tokeninfo is now deprecated

### CODE REVIEW

File: src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java [hunk 1]
```diff
@@ -246,12 +249,13 @@ public boolean hasProject(String projectID, String organizationId) {
     public boolean canAdministerApplication() {
         // NOTE: user can administer application by default if running without authentication
         if (Auth0Connection.authDisabled()) return true;
+        if (app_metadata == null) return false;
+        if (app_metadata.getDatatoolsInfo() == null)
```
Comment (bonjourmauko) [inline src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java]: What happens here if `app_metadata.getDatatoolsInfo() == null` ?

File: src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java [hunk 2]
```diff
@@ -246,12 +249,13 @@ public boolean hasProject(String projectID, String organizationId) {
     public boolean canAdministerApplication() {
         // NOTE: user can administer application by default if running without authentication
         if (Auth0Connection.authDisabled()) return true;
+        if (app_metadata == null) return false;
+        if (app_metadata.getDatatoolsInfo() == null) 
+        if(app_metadata.getDatatoolsInfo().permissions == null) return false;
```
Comment (maukoquiroga) [inline src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java]: Space missing after if.

File: src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java (hunk 1 above)
Comment (l-vincent-l) [reply src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java]: you're right I forgot a return false

**Review by bonjourmauko — APPROVED** [review]

File: src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java (hunk 1 above)
Comment (l-vincent-l) [reply src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java]: Done

**Pull request merged**